### PR TITLE
update wagtail to v4.2.4

### LINKED ISSUE #12: Minutes share
- live share for minutes (for people that attend the meeting virtually and want to follow the minutes)
- https://github.com/KalobTaulien/wagtail-livepreview

### LINKED ISSUE #13: Site preview in editing mode
1327 allowed the user to quickly preview the page while editing it. In Wagtail, this requires saving the draft and clicking the preview button, which opens a new tab. It would  be convenient to have a preview inside of the editor.

see #12 for an extension that offers this functionality

### PR DESCRIPTION
**Pull request by jeriox**
closes #13 closes #12

the issue found by @Paula-Kli back in the day seems to be gone, entering participants works fine for me although a bit CSS tweaking to bring it back to full width would be good 
grafik

### CODE REVIEW

**Review by Paula-Kli — COMMENTED** [review]
When having no participants that are recommended I still experience the same problem
image

The question that arises to me is whether that is ever the case in production

**Review by frcroth — APPROVED** [review]

**Review by frcroth — COMMENTED** [review]
Can you verify that select2 still works? For me it does not work (throw 404s)?

**Comment by jeriox** [comment]
> Can you verify that select2 still works? For me it does not work (throw 404s)?

works for me

File: myhpi/core/markdown/extensions.py [hunk 1]
```diff
@@ -140,6 +143,24 @@ def decrease(self, match):
...
+            el.text = util.AtomicString(_("[missing link]"))
+        return el, m.start(0), m.end(0)
+
+    def url(self, id):
+        return Page.objects.get(id=id).localized.get_url()
+
+    @staticmethod
+    def default_pattern():
+        return r"\[(?P<title>[^\[]+)\]\(page:(?P<id>\d+)\)"
+
```
Comment (frcroth) [inline myhpi/core/markdown/extensions.py]: Please readd the test then

**Review by frcroth — COMMENTED** [review]
:shipit:

**Pull request merged**